web/plugin-manager/src/components/pluginBlock.tsx

```diff
@@ -0,0 +1,183 @@
+import { SetStateAction, useMemo, useState } from "react";
+import { ArrowPathIcon, TrashIcon, PlayCircleIcon } from "@heroicons/react/24/outline";
+import axiosServices from "../services/axios";
+import { AxiosResponse } from "axios";
+import { Plugin, PluginProps, PluginStatus } from "../../../shared/interfaces/IPlugin";
+import { statusHelper } from "../helpers/statusHelpers";
+
+function PluginBlock(p: PluginProps) {
+    // Callback to set error on parent
+    function sendErrorData(errorType: string, errorMessage: string) {
+        p.setErrorData(errorType, errorMessage);
+    }
+    // Data to display
+    let dataMemoized = useMemo(() => {
+        console.log("p.props useMemo", p.props);
+        return p.props;
+    }, [p.props]);
+
+    // Toggle status state
+    const [toggleStatus, setStatus] = useState(statusHelper(p.props.status));
+
+    const [playStatusClassName, setPlayStatusClassName] = useState(playStatus());
+
+    // fetch info from plugin to get fresh data on demand
+    async function fetchPluginStatus(): Promise<AxiosResponse<string>> {
+        let result: AxiosResponse<string> = {} as AxiosResponse<string>;
+        try {
+            return (result = await axiosServices.getpluginInfo(dataMemoized.id));
+        } catch (error) {
+            sendErrorData(
+                "error",
+                `Plugins infos failed to get infos from plugin name : ${dataMemoized.name} on id: ${dataMemoized.id}`
```

Comment:
You're not getting information from the error content ?

Reply:
The goal was to have readable data. and to have a good ui.
If i display error.message it looks like shit

Reply:
Sounds good, but how can we access the error to debug if needed ? Are they logged in the console ? Should we plan a king of collapsable area to get additionnal information ? depending on the solution you image, can be done in another issue :-)

Reply:
Messages was personnalized on each error. So you can find it in the code. It was pretty same has we have in Thyra, except they are not in library. 

but i changed and now we display error.message.
And we will see in the future if we want to build something else. 

But for me in any case User don't have to know if error was 400 or 404, If he want to see this he can open the console and inspect the request. But Madame Michu doesn't care about this and if she need a debug she can copy the message and send it to us.